go.mod

```diff
@@ -23,7 +20,9 @@ replace github.com/ethereum-optimism/optimism => github.com/Layr-Labs/optimism v
 replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101503.2-rc.5
 
 require (
-	github.com/Layr-Labs/eigenda-proxy/clients v1.0.1
+	// TODO: update this to correct version after we've tested that it works
+	// should we use 0.2.1 which is the current eigenda-proxy/clients module version?
+	github.com/Layr-Labs/eigenda/api/proxy/clients v0.1.0
```

Comment:
I've tested this and it works. Just need to decide which version we want to publish:
- restart @ 0.1.0 ?
- 0.2.1 to keep same version as previous module?
- 2.1.0 to match the new monorepo release version?

cc @ethenotethan

Reply:
It sounds like @ethenotethans suggestion might be to bundle the release of the `clients` in with all the other releases entirely. @samlaf, you don't think that would work?

Reply:
hmm, good question. I guess it could work. I was assuming cannot work because its not part of the same go module, but that doesn't mean we can't still bundle them into a single release with matching version numbers..

Reply:
Getting back to this. Don't think it makes sense, because most releases won't have any proxy client changes, so doesn't make sense to tag a release randomly imo. A release for a golang module is simply pushing a tag, so it's very easy to do and we can do it whenever needed. I think I'm in favor of keeping this module separate and just bumping it whenever we need it.